.github/workflows/benchmark.yml

```diff
@@ -50,6 +50,7 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Compile
+        timeout-minutes: 15
```

Comment:
Was this the issue? That's unsettling

Reply:
No, I tried this to see if it helped the CI, but it didn't help...